packages/terra-date-picker/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Unreleased
 * Update tests for dev-site v6
 * Explicitly pull in Error icon from terra-icon instead of pulling from terra-form-field
 
+### Added
+* Aria-Label and translations added to input to fix accessibility issue of Date-Time-Picker.
+
```

Comment:
You're going to want to move this into the Unreleased section above.

Reply:
Moved in 3f1eea1